src/maingoblint.ml

```diff
@@ -433,8 +433,8 @@ let check_arguments () =
   if get_bool "ana.base.context.int" && not (get_bool "ana.base.context.non-ptr") then (set_bool "ana.base.context.int" false; warn "ana.base.context.int implicitly disabled by ana.base.context.non-ptr");
   (* order matters: non-ptr=false, int=true -> int=false cascades to interval=false with warning *)
   if get_bool "ana.base.context.interval" && not (get_bool "ana.base.context.int") then (set_bool "ana.base.context.interval" false; warn "ana.base.context.interval implicitly disabled by ana.base.context.int");
-  if get_bool "incremental.only-rename" then (set_bool "incremental.load" true; warn "incremental.only-rename implicitly activates incremental.load. Previous AST is loaded for diff and rename, but analyis results are not reused.")
-
+  if get_bool "incremental.only-rename" then (set_bool "incremental.load" true; warn "incremental.only-rename implicitly activates incremental.load. Previous AST is loaded for diff and rename, but analyis results are not reused.");
+  if get_bool "incremental.restart.sided.enabled" && get_string_list "incremental.restart_globs.globs" <> [] then (warn "Passing a non-empty list to incremental.restart_globs.globs (manual restarting) while incremental.restart.sided.enabled (automatic restarting) is activated. Aborting."; exit 5 )
```

Comment:
>Is it actually incompatible though? You could still manually restart globals that the change-based sided did not.

`destabilize_leaf` calls `destabilize`, which is changed to `destabilize_with_side`, in case that `restart_sided` is activated.  This changes the behavior of the manual restarting so that it restarts more. Of course this could be changed, but it seems to me that one would usually not want to have both the automatic and the manual restarting. And adding code for that case that resets the `destabilize` back and forth would make the code a bit more convoluted.

I guess instead of aborting, one could just give a warning here.